environs/manual/sshprovisioner/package_test.go

```diff
@@ -0,0 +1,10 @@
+// Copyright 2013 Canonical Ltd.
+// Licensed under the AGPLv3, see LICENCE file for details.
+
+// +build linux
+package sshprovisioner_test
+
+import stdtesting "testing"
+
+func Test(t *stdtesting.T) {
```

Comment:
This needs to run the gocheck tests. Why don't you just copy suite_test.go over as it was, and rename it to package_test.go?